src/main/java/com/wolox/trainnerInMaven/models/DTOs/PublisherDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.wolox.trainnerInMaven.models.DTOs;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.io.Serializable;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PublisherDTO implements Serializable {
```

Comment:
What  do you think about encapsulating PublisherDTO and AuthorDTO in one DTO? I think we can avoid redundancies.